autoload/clap/installer.vim

```diff
@@ -7,7 +7,7 @@ set cpoptions&vim
 let s:plugin_root_dir = fnamemodify(g:clap#autoload_dir, ':h')
 
 function! s:run_term(cmd, cwd, success_info) abort
-  10new belowright bottom
+  belowright 10new bottom
   setlocal buftype=nofile winfixheight norelativenumber nonumber bufhidden=wipe
```

Comment:
Is this one a typo? Because `10new belowright` will actually create a window whose name is "belowright bottom".